common/protob/messages-management.proto

```diff
@@ -427,31 +441,37 @@ message EntropyAck {
  * @next WordRequest
  */
 message RecoveryDevice {
```

Comment:
Since we are already renaming fields in the device recovery protobuf definitions and since connect will need work to adjust to these changes, could we also fix the typo in `RecoveryDevice` -> `RecoverDevice`? It has been irritating me for years. I don't want to hold up this review with unnecessary changes, but it seems like the perfect moment right now.